### PR TITLE
display step explanation as info box

### PR DESCRIPTION
**Pull request by paulapreuss**
Explanation text boxes through info messages instead of just having the text in the template

### CODE REVIEW

**Comment by paulapreuss** [comment]
. More visible, but I'm not sure how to make it so that the user can disable them if they get annoying?

**Comment by Bachibouzouk** [comment]
> . More visible, but I'm not sure how to make it so that the user can disable them if they get annoying?

We can do this in a second step, we can create a user_preferences table and in the view only set the message if the user has the correct setting. We would need this only in a second step and not for the draft

File: app/templates/cp_nigeria/steps/step_progression.html [hunk 1]
```diff
@@ -43,6 +43,14 @@ <h1 class="wizard__name">
 
         <div class="step_description">
             {% block step_explanation %}
+					 {% if messages %}
```
Comment (Bachibouzouk) [inline app/templates/cp_nigeria/steps/step_progression.html:46]: This makes use of messages in the template @paulapreuss

File: app/templates/cp_nigeria/base_cpn.html [hunk 2]
```diff
@@ -68,14 +68,6 @@
 {% endblock index %}
 
 
-<!-- modal to show debug messages -->
```
Comment (Bachibouzouk) [inline app/templates/cp_nigeria/base_cpn.html:71]: This was creating the modals for the messages @paulapreuss

**Comment by Bachibouzouk** [comment]
closes #5

**Pull request merged**